.travis.yml

```diff
@@ -12,13 +12,25 @@ cache:
   directories:
       - $HOME/cachedir
 
+services:
+  - mysql
+
 php:
     - 7.1
 
+matrix:
```

Comment:
Any reason we need this new section included? Cannot the 3 new xenial runs go to the ```env``` one and forget? I personally find things more readable if together. Not critical though.

Reply:
No, you can't change disto using env definition, only option that should work in theory is (with leaving env as is):
```
dist:
 - trusty
 - xenial
```
But it does not work that way, so had to use matrix instead.
(that DIST var is dummy, it is only to see which distro was used when you are on Travis build page)